src/v2/Apps/Auction/Routes/__fixtures__/MutationResults/createBidderPosition.ts

```diff
@@ -45,8 +45,38 @@ export const createBidderPositionFailed: ConfirmBidCreateBidderPositionMutationR
   createBidderPosition: {
     result: {
       position: null,
-      status: "FAILED",
```

Comment:
Not sure where this `FAILED` status came from but [we never return it from MP](https://github.com/artsy/metaphysics/blob/d6257e33f6eccf7bc9e521baffd55a7817f8f8ad/src/schema/v2/me/bidder_position_messages.ts#L5-L10) so I simply replaced it with `SALE_CLOSED` which seems more appropriate.